openslides/__main__.py

```diff
@@ -165,12 +173,23 @@ def start(args):
 
     execute_from_command_line(['manage.py', 'migrate'])
 
+    # Set host and port
+    if args.host:
+        host = args.host
+    else:
+        host = "0.0.0.0"
+    if args.port:
+        port = args.port
+    else:
+        port = "8000"
+
+    # Open browser always on localhost (0.0.0.0 is not supported on Windows)
     if not args.no_browser:
-        start_browser('http://localhost:8000')
+        start_browser('http://localhost:%s' % port)
```

Comment:
The browser should open at the given host. Only if it is `0.0.0.0`, it should be changed to `localhost`.